angular_gallery_section/lib/builder/gallery_section_builder.dart

```diff
@@ -25,24 +26,41 @@ class GallerySectionBuilder extends Builder {
 
     final mergedImports = new Set<String>();
     final mergedDemos = <String, String>{};
+    final apis = [];
```

Comment:
`apis` will have type `List<dynamic>`. Maybe provide a type argument to the array constant?